references/cli/dryrun.go

```diff
@@ -52,7 +53,8 @@ func NewDryRunCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command
 
 			parser := appfile.NewApplicationParser(newClient, dm)
 
-			appFile, err := parser.GenerateAppFile(app.Name, app)
+			// TODO(wangyike) this context only for compliing, in the future maybe we will support cli?
+			appFile, err := parser.GenerateAppFile(context.TODO(), app.Name, app)
```

Comment:
I think we can use namespace of this app in the context

Reply:
Yes, updated